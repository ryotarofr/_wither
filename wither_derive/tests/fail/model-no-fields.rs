use serde::{Serialize, Deserialize};
use wither::Model;

#[derive(Serialize, Deserialize, Model)]
struct BadModel;

fn main() {}
