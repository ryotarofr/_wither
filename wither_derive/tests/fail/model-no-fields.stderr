error: wither models must have named fields
 --> $DIR/model-no-fields.rs:5:1
  |
5 | struct BadModel;
  | ^^^^^^^^^^^^^^^^
